deluge/ui/hostlist.py

```diff
@@ -163,12 +163,12 @@ def get_host_info(self, host_id):
             host_id (str): The host id to get info on.
 
         Returns:
-            list: A list of (host_id, hostname, port, username).
+            list: A list of (host_id, hostname, port, username, password).
```

Comment:
It's a bit of a red flag to return a cleartext password to solve this issue. I think the correct solution is to use `hostlist.connect_host` in `json_api.stop_daemon`